google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryImpl.java

```diff
@@ -1125,8 +1210,20 @@ public TableDataInsertAllResponse call() throws Exception {
                 getOptions().getClock());
       } catch (RetryHelperException e) {
         throw BigQueryException.translateAndThrow(e);
+      } catch (IllegalArgumentException | IOException e) {
+        // Invalid universe domain exceptions.
+        throw BigQueryException.translateAndThrow(e);
       }
     } else {
+      // This calls the backend without the retry wrapper so first validate the universe domain.
```

Comment:
for the purposes of standardizing the pattern here, is it worth using validateAndRunWithRetries here, but passing in a static retry config that only allows a single invocation rather than using `getOptions().getRetrySettings()`?